### PR TITLE
Added support for "exact" into Route

### PR DESCRIPTION
**Pull request by tiberiuc**
"exact" added to Route to match exact route path in nested routes

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ function segmentize(uri) {
  * @return {string}
  */
 function stripSlashes(str) {
+  console.log('---',str)
```
Comment (mazipan) [inline src/utils.js]: remove log

**Comment by EmilTholin** [comment]
Hi @tiberiuc!

Thank you for the PR, but may I ask why you need an `exact` prop? This library takes the Reach Router approach of deriving the best `Route` match in the `Router`. If you are having some issues, there might be some other way to go about it or solve it than adding an `exact` prop.

**Comment by tiberiuc** [comment]
Hi,

sorry for late response. I had one component that is a wrapper for tree different pages which respond on three different path not inside the same sub-path ( like /login / register / forgot-password )

there are a few options to have that:
1) put them inside thew same path like /auth/login ,  /auth/register ... etc
2) create wrapper components for each page 
3) use "exact" prop so router don't match it as a sub-path

I find it that using "exact" prop it is more elegant because the first option is not really an option some time and creating another top level components just add some unnecessary bloatware into the code.

Let me know if you have any other options

File: src/utils.js (hunk 1 above)
Comment (tiberiuc) [reply src/utils.js]: thank you, my mistake, I fixed it

**Comment by EmilTholin** [comment]
Hi @tiberiuc!

Thanks, that makes a lot of sense. We opted for removing the [implicit`/*` wildcard from the `Route` path](https://github.com/EmilTholin/svelte-routing/issues/18) so that an `exact` prop won't be needed.

**Comment by tiberiuc** [comment]
I tested it with the latest version and the example I gave you it still not work because not the '*' was the issue but appending subroutes. Maybe another solution instead 'exact' prop is to have something like this: in subroutes, when the defined route is starting with '/' then not append to parent route otherwise append to parent route ( the way it is working now). I don't know how a change like that break existing projects

**Pull request closed without merging**